index.js

```diff
@@ -13,25 +13,37 @@ module.exports = {
 
   included() {
     let app = findHost(this);
+    let desiredComponents = [
+      'apacheconf',
+      'bash',
+      'css',
+      'handlebars',
```

Comment:
we don't want `handlebars` -- because we don't use this syntax (see the removed list, it's not present)

Reply:
The Git history is correct - I built from after your change. but, I copy/pasted a chunk of code from where I was debugging it over in the guides, which hadn't picked up your change yet. Sorry about that. I went back and double-checked everything else in this file and it doesn't look like I missed anything further.